battlebots/database/access.py

```diff
@@ -37,7 +37,8 @@ def remove_bot(user, botname):
                         % (user.nickname, botname, code_dir))
         pass
 
-    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
```

Comment:
@procrat Yes! Good thinking, that should indeed be the case and fix this
problem
On 8 Apr 2016 11:26, "Stijn Seghers" <notifications@github.com> wrote:

> In battlebots/database/access.py
> <https://github.com/ZeusWPI/aichallenge/pull/85#discussion_r59000580>:
>
> > @@ -37,7 +37,8 @@ def remove_bot(user, botname):
> >                          % (user.nickname, botname, code_dir))
> >          pass
> >
> > -    bot = session.query(Bot).filter_by(user=user, name=botname).one_or_none()
>
> @Silox <https://github.com/Silox>: I just reread your first comments. Why
> wouldn't there be an open session? If we chose (2), than either the ranker
> or the web server should have opened one. Otherwise this code shouldn't be
> called in the first place. Right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ZeusWPI/aichallenge/pull/85/files/1478b56574b04b5a61ef6433eb7d796aa0f7b628#r59000580>
>